src/camlsnark_c/libsnark-caml/README.md

```diff
@@ -16,7 +16,7 @@ For announcements and discussions, see the [libsnark mailing list](https://group
 [TOC]
```

Comment:
this is part of libsnark, could you open a PR to fix the typos there?

Reply:
https://github.com/scipr-lab/libsnark/pull/128